src/components/Button.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+
+
+
+const Button = ({ type = 'button', onClick, children }) => {
+    return (
+        <button type={type} onClick={onClick} >
+            {children}
+        </button>
+    );
+};
```

Comment:
Dit component is nu een doorgeefluik. Zelf zou ik hier niet een apart component voor maken, maar gewoon direct het html button element gebruiken. Het zorgt nu niet perse voor code reductie namelijk